implementation/registry-of-identifiers/src/main/java/eu/efti/identifiersregistry/entity/Consignment.java

```diff
@@ -0,0 +1,72 @@
+package eu.efti.identifiersregistry.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+@Table(name = "consignment")
+public class Consignment implements Serializable {
+
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    @Column(name = "id")
+    private long id;
+    @Column(name = "platform_id")
+    private String platformId;
+    @Column(name = "dataset_id")
+    private String datasetId;
+    @Column(name = "gate_id")
+    private String gateId;
+    @Column(name = "carrier_acceptance_datetime")
+    private OffsetDateTime carrierAcceptanceDatetime;
+    @Column(name = "delivery_event_actual_occurrence_datetime")
+    private OffsetDateTime deliveryEventActualOccurrenceDatetime;
+
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER, orphanRemoval = true, mappedBy = "consignment")
+    private List<MainCarriageTransportMovement> mainCarriageTransportMovements = new ArrayList<>();
+
+    public void setMainCarriageTransportMovements(List<MainCarriageTransportMovement> mainCarriageTransportMovements) {
```

Comment:
temporary too?

Reply:
mainCarriageTransportMovements is a part of the identifiers dataset, so it should be included.

Reply:
yes, i was  surprise by the code needed for setMainCarriageTransportMovements and setUsedTransportEquipments. I think i get why but isnt a @OneToMany with a JoinColumn enough to manage this? 

But anyway once the dto match the model you will be able to map automaticly the list and this code will not be needed anymore.